### PR TITLE
Mase

### PR DESCRIPTION
**Pull request by j1c**
#### What does this implement/fix? Explain your changes.
Changes:
1. Implement `MultipleASE`
1. Implement `BaseEmbedMulti` for multiple graph embedding methods
2. Add `copy` arg to `import_graph` so that we don't duplicate the data for `OmnibusEmbedding` and `MultipleASE` which is extremely costly

### CODE REVIEW

**Comment by j1c** [comment]
Ready

File: graspy/embed/mase.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+            best_dimension = self.n_components
+
+        if not self.scaled:
+            Us = np.hstack([U[:, :best_dimension] for U in Us])
+            Vs = np.hstack([V.T[:, :best_dimension] for V in Vs])
+        else:
+            # Equivalent to ASE
+            Us = np.hstack(
+                [
+                    U[:, :best_dimension] @ np.diag(D[:best_dimension])
```
Comment (bdpedigo) [inline graspy/embed/mase.py]: should these be sqrt(D)

File: pytest.ini [hunk 2]
```diff
@@ -7,3 +7,4 @@ filterwarnings =
     ignore:the matrix subclass is not
     ignore:Using a non-tuple
     ignore:Input graph is not fully connected.
+    ignore:Number of distinct clusters
```
Comment (bdpedigo) [inline pytest.ini]: what is this from

File: graspy/embed/mase.py (hunk 1 above)
Comment (j1c) [reply graspy/embed/mase.py]: Yes

File: pytest.ini (hunk 2 above)
Comment (j1c) [reply pytest.ini]: This is from GMM. Realized there is no randomness in my tests so changed the undirected version to the one in the paper, but changed the probability values for directed case to be something different than 1. 

Removed this line.

**Comment by j1c** [comment]
Maybe `scores_` -> `graph_scores_` or `individual_scores_`? I do not want to deviate from the wording from the paper too much.

**Pull request merged**